.github/workflows/build.yml

```
@@ -27,8 +27,11 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        zephyr-version: ["4.2", "lts"]
```

```suggestion
    strategy:
      matrix:
        zephyr:
          - version: "4.2"
            testcase: "astarte.default"
          - version: "lts"
            testcase: "astarte.lts"
```
I'm not talking about adding a third component, you can do it also like this, and you also make clear by looking at the matrix the `lts` will use the `astarte.lts` testcase.